xdc-plugin/tests/minilitex_ddr_arty.tcl

```diff
@@ -0,0 +1,15 @@
+yosys -import
```

Comment:
Can you put this test in it's own directory

Reply:
Agreed

Reply:
Sure, I moved all of the tests to separate directories.